### PR TITLE
QDEV-687 Public API for searching with Qase Query Language

### PR DESCRIPTION
**Pull request by abelharisov**

### CODE REVIEW

**Review by kamilsk — CHANGES_REQUESTED** [review]
please rollback code generation part.

motivation: we have to focus on the most valuable part of the pull request - specification changes. you can generate examples a little bit later, then your pr will be approved, for example.

**Comment by abelharisov** [comment]
> please rollback code generation part.
> 
> motivation: we have to focus on the most valuable part of the pull request - specification changes. you can generate examples a little bit later, then your pr will be approved, for example.

done

File: src.yaml [hunk 1]
```diff
@@ -35,19 +36,27 @@ servers:
...
     Token: { $ref: './headers/Token.yaml' }
 
   responses:
     Response: { $ref: './responses/Response.yaml' }
     ProjectListResponse: { $ref: './responses/ProjectListResponse.yaml' }
+    SearchResponse: { $ref: './responses/SearchResponse.yaml' }
 
   parameters:
     Limit: { $ref: './parameters/pagination/Limit.yaml' }
     Offset: { $ref: './parameters/pagination/Offset.yaml' }
```
Comment (kamilsk) [inline src.yaml:61]: what do you think if we rename them to PageLimit, PageOffset?

**Review by kamilsk — COMMENTED** [review]
LGTM

minor changes are needed, but we can discuss them outside the pr

**Review by kamilsk — APPROVED** [review]

**Comment by kamilsk** [comment]
please do not include generate part, we do it later. maybe we extract it from the specs

**Pull request merged**